codes/quantum/qubits/monitored_random_circuits.yml

```diff
@@ -0,0 +1,94 @@
+# that entry instead of here.
+description: 'A monitored random circuit is described by a series of intermittant local projective Pauli measurements with unitary time evolution operators. When the rate of projective measurements is independently controlled by a probability parameter p, there exist two sable phases, one described by the volume-law entanglement entropy and the other described by area-law entanglement entropy. The phase transition can be understood from the perspective of quantum error correction and information scrambling, as discussed in \cite{arXiv:1903.05124}. In the volume-law phase, the random unitaries scramble the information (rotate the code words), protecting the information from the projective measurements (aka errors). When the rate of measurements surpasses a threshold \( p_c \) the information is no longer protected. The key insight offered by the quantum error correction implemented by the scrambling dynamics is that the entanglement and correlation between subsystems A, B is scrambled into non-local degrees of freedom and cannot be disentangled by local measurements below \( p < p_c \).'
+
+- '\textbf{Clifford monitored random circuits}: When considering unitary evolutions, it can be useful to only condier unitaries that belong to the Clifford group such that the series of time evolved Pauli operators are always Pauli operators, by definition of the Clifford group. In this case, the time-evolved projective measurement at time step $j$ is given by \( P_j = U_{\tau} \cdots U_j P_j^’ (U_{\tau} \cdots U_j)^{\dagger} \). Thus the monitored random circuit just implements a series of projective measurements at various time steps.'
+
+protection: 'When in the volume-law phase, protects against random projective measurements at a rate \( p < p_c \). As described in \cite{1903.05124} and \cite{2105.13352} the subleading contribution to the entropy can be interpreted as the code distance and is given by \(d_{\rm code} = L^{\gamma_{code}} \) where \( \gamma_{code} \approx 0.38 \) and \( L \) is the size of the system. For a monitored circuit in the volume law-phase with an exponentially long memory time, \( d \leq Poly(n) \).'
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Finite rate \cite{arXiv:2007.03822}. Can think of as a stabilizer code whose parameters depend on time, \( [[n,k(t),d(t)]] \)'
```

Comment:
The correct reference is 1905.05195.  Also applies for Haar random circuits with measurements (see 1911.00008)